flow-typed/box-ui-elements.js

```diff
@@ -688,29 +691,38 @@ type ErrorResponseData = {
 
 type ElementsXhrError = $AxiosError<any> | ErrorResponseData;
 
-type ErrorOrigins =
-    | ORIGIN_CONTENT_SIDEBAR
-    | ORIGIN_CONTENT_PREVIEW
-    | ORIGIN_PREVIEW
-    | ORIGIN_DETAILS_SIDEBAR
-    | ORIGIN_ACTIVITY_SIDEBAR
-    | ORIGIN_SKILLS_SIDEBAR
-    | ORIGIN_METADATA_SIDEBAR;
+type ElementsOrigins =
+    | typeof ORIGIN_CONTENT_SIDEBAR
+    | typeof ORIGIN_CONTENT_PREVIEW
+    | typeof ORIGIN_PREVIEW
+    | typeof ORIGIN_DETAILS_SIDEBAR
+    | typeof ORIGIN_ACTIVITY_SIDEBAR
+    | typeof ORIGIN_SKILLS_SIDEBAR
+    | typeof ORIGIN_METADATA_SIDEBAR;
 
 type ElementsError = {
     type: 'error',
     code: string,
     message: string,
-    origin: ErrorOrigins,
+    origin: ElementsOrigins,
     context_info: Object,
 };
 
 type ErrorContextProps = {
-    onError: (error: ElementsXhrError | Error, code: string, contextInfo?: Object, origin: ErrorOrigins) => void,
+    onError: (error: ElementsXhrError | Error, code: string, contextInfo?: Object, origin: ElementsOrigins) => void,
 };
 
 type ElementsErrorCallback = (e: ElementsXhrError, code: string, contextInfo?: Object) => void;
 
 type ClassificationInfo = {
     Box__Security__Classification__Key?: string,
 } & MetadataInstance;
+
+type MetricTypes =
```

Comment:
This type represents a singular item, so `MetricType` seems a more appropriate name.